Atomic/rpm_host_install.py

```diff
@@ -25,6 +26,10 @@ def copyfile(src, dest):
             return True
         else:
             shutil.copy2(src, dest)
```

Comment:
It's more robust to use `setfscreatecon()` to atomically create the file with the right context; see what `ostree-sepolicy.c` does.

Reply:
thanks for the hint, I'll give it a try and update the patch.  I'll need to check the "copy2" doesn't override it.